[TASK] Use permalinks to docs.typo3.org

linawolf opened the pull request:
These links will keep working even if the content should be moved in the future.

linawolf committed:
[TASK] Use permalinks to docs.typo3.org

These links will keep working even if the content should be moved in the future.

andreaskienast commented on templates/default/partials/version/system-requirements.html.twig:
```diff
@@ -1,5 +1,6 @@
 {% frame with { id: 'system-requirements', color: 'lighter', title: 'System Requirements' } %}
-    <p>For more information and installation instructions read the <a href="https://docs.typo3.org/m/typo3/tutorial-getting-started/master/en-us/SystemRequirements/Index.html" rel="noopener,noreferrer">Installation guide</a>.</p>
+    <p>For more information and installation instructions read the
+        <a href="https://docs.typo3.org/permalink/t3coreapi:system-requirements" rel="noopener,noreferrer">Installation guide</a>.</p>
```
`noreferrer` already implies `noopener`: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/rel/noreferrer

However, `noopener` should be sufficient.

andreaskienast reviewed (commented):
While changing the URLs, please add `rel="noopener"` where missing.

linawolf commented:
@andreaskienast I added it where it was missing and removed noreferrer where it was still found. Additionally added target="_blank" where it was still missing
